bp_common/src/include/bp_common_cache_service_if.vh

```diff
@@ -68,7 +79,9 @@ typedef enum logic [1:0] {
  // read cache block
  e_cache_data_mem_read,
  // write uncached load data
- e_cache_data_mem_uncached
+ e_cache_data_mem_uncached,
+ // write L2 atomic data
+ e_cache_data_mem_amo
```

Comment:
Can reuse data_mem_uncached.  After all, from the L1 perspective, L2 atomics are "uncached"